docs/tutorial/context-isolation.md

```diff
@@ -0,0 +1,70 @@
+# Context Isolation
+
+## What is it?
+
+Context Isolation is a feature that ensures that both your `preload` scripts and Electron's internal logic run in a separate context to the website you load in a `webContents`.  This is important for security purposes as it ensures that the website can't access Electron internals or the powerful APIs your preload script has access to.
+
+This means that the `window` object that your preload script has access to is actually a **different** object than the website would have access to.  If you set `window.hello = 'wave'` in your preload script and context isolation is enabled `window.hello` will be undefined if the website tries to access it.
+
+Every single application should have context isolation enabled and from Electron 12 it will be enabled by default.
+
+## How do I enable it?
+
+From Electron 12 it will enabled by default, for lower versions it is an option in the `webPreferences` option when constructing `new BrowserWindow`'s.
```

Comment:
```suggestion
From Electron 12, it will be enabled by default. For lower versions, it is an option in the `webPreferences` option when constructing `new BrowserWindow`'s.
```